### PR TITLE
[6265bis] Specify that no decoding of Set-Cookie line should occur

### LINKED ISSUE #1517: RFC 6265bis should specify that no decoding of Set-Cookie line should occur
UAs should not decode any URL-encoded (or otherwise encoded) characters while storing a cookie, e.g. a cookie such as `__%53ecure-foo=bar` should not be interpreted as a `__Secure-` cookie.

I landed some WPTs for this in https://github.com/web-platform-tests/wpt/pull/28066. Chrome, Edge, Firefox, and Safari [seem to](https://wpt.fyi/results/cookies/attributes/secure-non-secure.html?label=experimental&label=master&aligned) do the right thing.

I think this could just be a clarifying note in the parsing algorithm section.

### PR DESCRIPTION
**Pull request by johannhof**
Fixes #1517

### CODE REVIEW

File: draft-ietf-httpbis-rfc6265bis.md [hunk 1]
```diff
@@ -1131,6 +1131,11 @@ explicitly. Horizontal tab (%x09) is excluded from the CTL characters that
 lead to set-cookie-string rejection, as it is considered whitespace, which is
 handled separately.
 
+NOTE: The set-cookie-string may contain octet sequences that appear
+percent-encoded as per {{Section 2.1 of RFC3986}}. However, a user agent
```
Comment (chlily1) [inline draft-ietf-httpbis-rfc6265bis.md:1135]: I think the section number goes outside of the {{}}.

Comment (chlily1) [inline draft-ietf-httpbis-rfc6265bis.md:1135]: This should be added to the informative references at the top.

**Comment by DiamondG3m** [comment]
```

```~~

File: draft-ietf-httpbis-rfc6265bis.md (hunk 1 above)
Comment (johannhof) [reply draft-ietf-httpbis-rfc6265bis.md:1135]: With `{{Section 2.1 of RFC3986}}` it creates an additional deeplink into that section:
image

Which I think is nice, so I'll keep it like that if you don't mind :)

Comment (johannhof) [reply draft-ietf-httpbis-rfc6265bis.md:1135]: I think it's already there, but maybe I'm missing something.

Comment (chlily1) [reply draft-ietf-httpbis-rfc6265bis.md:1135]: TIL, that's great. :)

Comment (chlily1) [reply draft-ietf-httpbis-rfc6265bis.md:1135]: Nope, I skimmed too quickly. My bad!

**Comment by chlily1** [comment]
Thanks! This Lgtm but I'll let Steven take a look too

**Comment by sbingler** [comment]
This looks to me as well, thanks!

**Pull request merged**